data_seeder/seeder.py

```diff
@@ -0,0 +1,303 @@
+import requests
+from pathlib import Path
+import json
+import os
+import urllib3
+import xml.etree.ElementTree as ET
+import random
+from jinja2 import Environment, FileSystemLoader
+from alive_progress import alive_bar
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+DATA_DIR = Path(__file__).parent.parent / "data"
+DATA_JSON_FILENAME = "data.json"
+XML_TEMPLATES_DIRNAME = Path(__file__).parent / "xml_templates"
+
+API_KEY = os.environ["PRESTASHOP_API_KEY"]
+
+IP_PORT = "https://localhost:8443"
```

Comment:
Can we use an environment variable here, with localhost as a fallback value?

Reply:
Sure